main.js

```diff
@@ -501,8 +561,7 @@
   }
 
   function NextSectionLink(props) {
-    if (props.index >= dswContent.length - 1)
-    {
+    if (props.index >= dswContent.length - 1) {
```

Comment:
You might want to just remove the `NextSectionLink`s until they're fixed. They don't set the `activeMenuState` right now.

Reply:
When removing this section, it doesn't render.